### PR TITLE
Fix better type check for lecture blocks

### PR DESCRIPTION
**Pull request by hakjeri**
Still not ideal, but the Slides array will always be there, unlike the Document field

### CODE REVIEW

File: src/types/checkers.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
 export const isBlockOneLevelDeep = (
   data: Data<DownloadableContent>
 ): data is Data<BlockOneLevelDeep> =>
-  !!(data as Data<BlockOneLevelDeep>).attributes.Document
+  !!(data as Data<BlockOneLevelDeep>).attributes.Slides
```
Comment (tasola) [inline src/types/checkers.ts:12]: Why this change?

**Review by tasola — APPROVED** [review]

File: src/types/checkers.ts (hunk 1 above)
Comment (hakjeri) [reply src/types/checkers.ts:12]: It seemed `Document` does not always exist on Blocks, whereas the `Slides` array does

Comment (tasola) [reply src/types/checkers.ts:12]: Ah right, that's true! Nice
